node/connection_base.js

```diff
@@ -197,7 +197,8 @@ TChannelConnectionBase.prototype.resetAll = function resetAll(err) {
     outOpKeys.forEach(function eachOutOp(id) {
         var req = self.requests.out[id];
         delete self.requests.out[id];
-        // TODO: shared mutable object... use Object.create(err)?
+        err = Object.create(err);
+        err.connectionClosing = true;
```

Comment:
Will need to ensure that we use the good is-error implementation to handle this case. Might be better to clone properties and use a fresh Error instance for each. TypedError.clone(error)?